ui/app/css/itcss/components/transaction-list.scss

```diff
@@ -63,14 +63,22 @@
   justify-content: flex-start;
 
   @media screen and (max-width: $break-small) {
-    margin: 0 1.3em .95em;
+    padding: 0 1.3em .95em;
   }
 
   @media screen and (min-width: $break-large) {
     margin: 0 2.37em;
   }
 }
 
+.tx-list-pending-item-container {
+  cursor: pointer;
+}
```

Comment:
you can do `&:hover` in here to point to hover state. Sass will compile it for us.

Also, `alto` seems too strong for a hover color. Let's try `rgba($alto, .2)`.

Great call on adding UI cue for clickable area!

Reply:
Fixed in e860db00